src/web/rustdoc.rs

```diff
@@ -282,7 +301,10 @@ pub fn rustdoc_html_server_handler(req: &mut Request) -> IronResult<Response> {
         );
         let url = ctry!(req, Url::parse(&redirect_path));
 
-        Ok(super::cached_redirect(url, config.cache_rustdoc_redirects))
+        Ok(super::cached_redirect(
+            url,
+            config.cache_rustdoc_redirects_version,
```

Comment:
This is not always correct, for `MatchVersion::Semver` it should be `cache_crate`, not `cache_version`. Can you take this as a parameter instead?

Maybe we should consider changing the name of the parameter too :thinking: I think `cache_release` might be easier to think about than `cache_version`.

Reply:
@jyn514 I was already working on exactly this :) stumbled on the same thing. 

Parameter I'll rename.